src/main/java/com/spring/boot/project/demo/config/SecurityConfig.java

```diff
@@ -0,0 +1,17 @@
+package com.spring.boot.project.demo.config;
+
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+    protected void configure(HttpSecurity http) throws Exception {
+        http.formLogin();
```

Comment:
```suggestion
        http.formLogin()
            .authorizeRequests()
            ....
```
Можна зробити це одним pipeline

Reply:
Ок, переделаю)